doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,920 @@
+
+```{code-cell} ipython3
+c = 10.6
+
+X = Weibull(c=10.6)  
+Y = 1 / X  # compare to `invweibull`
+Y.plot()
+
+x = np.linspace(0.8, 2.05, 300)
+plt.plot(x, stats.invweibull(c=c).pdf(x), '--')
+plt.show()
+```
+
+`scipy.stats.chis2` describes a sum of the squares of normally-distributed random variables.
+
+```{code-cell} ipython3
+X = stats.Normal()
+Y = X**2  # compare to chi2
+Y.plot(t=('x', eps, 5));
+
+x = np.linspace(eps, 5, 300)
+plt.plot(x, stats.chi2(df=1).pdf(x), '--')
+plt.ylim(0, 3)
+plt.show()
+```
+
+`scipy.stats.foldcauchy` describes the absolute value of a Cauchy-distributed random variable. (A "folded" distribution is the distribution underlying the absolute value of a random variable.)
+
+```{code-cell} ipython3
+Cauchy = stats.make_distribution(stats.cauchy)
+c = 4.72
+
+X = Cauchy() + c  
+Y = abs(X)  # compare to `foldcauchy`
+Y.plot(t=('x', 0, 60))
+
+x = np.linspace(0, 60, 300)
+plt.plot(x, stats.foldcauchy(c=c).pdf(x), '--')
+plt.show()
+```
+
+`scipy.stats.lognormal` describes the exponential of a normally distributed random variable. It is so-named because the logarithm of the resulting random variable is normally distributed. (In general, a "log" distribution is typically the distribution underlying the *exponential* of a random variable.)
+
+```{code-cell} ipython3
+u, s = 1, 0.5
+
+X = stats.Normal()*s + u
+Y = stats.exp(X)  # compare to `lognorm`
+Y.plot(t=('x', eps, 9))
+
+x = np.linspace(eps, 9, 300)
+plt.plot(x, stats.lognorm(s=s, scale=np.exp(u)).pdf(x), '--')
+plt.show()
+```
+
+`scipy.stats.loggamma` describes the logarithm of of a Gamma-distributed random variable. Note that according to typical conventions, the appropriate name of this distribution would be [exp-gamma](https://reference.wolfram.com/language/ref/ExpGammaDistribution.html.en#:~:text=The%20term%20exp%2Dgamma%20is,of%20qualitatively%20similar%20probability%20distributions.) because the exponential of the RV is gamma-distributed.
+
+```{code-cell} ipython3
+Gamma = stats.make_distribution(stats.gamma)
+a = 0.414
+
+X = Gamma(a=a)  
+Y = stats.log(X)  # compare to `loggamma`
+Y.plot()
+
+x = np.linspace(-17.5, 2, 300)
+plt.plot(x, stats.loggamma(c=a).pdf(x), '--')
+plt.show()
+```
+
+`scipy.stats.truncnorm` is the distribution underlying a truncated normal random variable. Note that the truncation transformation can be applied either before or after shifting and scaling the normally-distributed random variable, which can make it much easier to achieve the desired result than `scipy.stats.truncnorm` (which inherently shifts and scales *after* truncation).
+
++++
+
+a, b = 0.1, 2
+
+X = stats.Normal()  
+Y = stats.truncate(X, a, b)  # compare to `truncnorm`
+Y.plot()
+
+x = np.linspace(a, b, 300)
+plt.plot(x, stats.truncnorm(a, b).pdf(x), '--')
+plt.show()
+
++++
```

Comment:
This should be a code cell.

Reply:
Weird... I prepared all this as a notebook. Maybe by accident I made the cell markdown instead of code after writing it.